expertsystem/data.py

```diff
@@ -3,11 +3,8 @@
 __all__ = [  # fix order in API
     "ParticleCollection",
     "Particle",
-    "ComplexEnergyState",
-    "QuantumState",
     "Parity",
     "Spin",
-    "ComplexEnergy",
     "create_antiparticle",
     "create_particle",
     "GellmannNishijima",
```

Comment:
This was added in order to (1) hide certain methods from the API and (2) fix the order of the definitions. It seems, however, that (2) fails somehow:
https://pwa--244.org.readthedocs.build/projects/expertsystem/en/244/api/expertsystem.data.html#module-expertsystem.data

Not sure why it doesn't work anymore since 11d36d37e028bdd6faf922cec741de76db7a1c3f (upgrade from Sphinx 2.4.4 to 3.2.1), see also "order" in
https://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html#directives

Reply:
I see, can you take a look at this?

Reply:
Yeah I checked after writing, don't have a good solution for it now. As this problem was there already before this PR, let's leave it as is.